.pre-commit-config.yaml

```diff
@@ -24,12 +24,11 @@ repos:
         - --exclude=binder/
         - --exclude=versioneer.py
 - repo: https://github.com/astral-sh/ruff-pre-commit
-  rev: v0.1.14
+  rev: v0.2.0
   hooks:
     - id: ruff
-      args: ["--fix", "--show-source"]
+      args: ["--fix", "--output-format=full"]
     - id: ruff-format
-      args: ["--line-length=100"]
```

Comment:
Note to self: this is redundant with the setting already in `pyproject.toml`.

Note to self 2: check the commit notes! :joy: